itou/users/admin.py

```diff
@@ -153,7 +153,7 @@ def queryset(self, request, queryset):
 class ItouUserAdmin(UserAdmin):
 
     form = UserAdminForm
-    inlines = UserAdmin.inlines + [
+    inlines = list(UserAdmin.inlines) + [
```

Comment:
Comme indiqué dans https://github.com/django/django/commit/dc9deea8e85641695e489e43ed5d5638134c15c7, l’intention est de “override” les `inlines` et non de les `mutate` (même s’il s’agit d’une shallow copy ici).
Comme de toute façon, le parent n’a pas de inlines, autant suivre la préférence de Django.
https://github.com/django/django/blob/bee09df39fe0734eb2388a2b3439436796592820/django/contrib/auth/admin.py#L44-L230

```suggestion
    inlines = [
```

Reply:
Fait !